api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -274,7 +155,7 @@ def publish_cdr(cmd_name, args)
   common = Common.new
   env = ENVIRONMENTS[op.opts.project]
   tier = env.fetch(:accessTiers)[op.opts.tier]
-  source_cdr_project = tier.fetch(:source_cdr_project)
+  source_cdr_project = env.fetch(:source_cdr_project)
```

Comment:
Will we always source multiple tiers from the same project?  For the previous implementation I assumed it was possible we wouldn't.

Reply:
For the foreseeable future, I think they'll be from the same project. We can push it back into the tier definition later, if needed.